web/testing/issue160/index.html

```diff
@@ -41,7 +41,7 @@
         attachType:'auto'
         });
       kmw.addKeyboards(
-        {id:'issue160',name:'Issue 160',languages:{id:'en',name:'English'}, filename:'./issue160-1.0.js'},
+        {id:'issue160',name:'Issue 160',languages:{id:'en',name:'English'}, filename:'./issue160.js'},
         {id:'tchaduni',name:'tchaduni',languages:{id:'fr',name:'TChad'}, filename:'./tchaduni-3.1.js'}
```

Comment:
The `tchaduni` keyboard didn't get included on the original PR #208. For cleanup, we either remove the reference or add the js file.

Reply:
@mcdurdin Any preference on which way this one is resolved?

Reply:
Not really. If there isn't a demonstrated need for the keyboard we can just turf it.